src/Mapbender/CoreBundle/Resources/translations/messages.fr.yaml

```diff
@@ -83,6 +83,7 @@ mb:
         description: Permet de créer un POI (point d'intérêt) pour le partager
       label:
         text: Text
+      accept: 'créer'
```

Comment:
upper case

Reply:
Has already been changed with the text correction "erstellen" to "Link erstellen".